packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -300,6 +333,10 @@ export default decorate([
         remote,
       }),
     },
+    computed: {
+      remoteWithInfo: (_, { remotes, remotesInfo }) =>
+        _remoteWithInfo(remotes, remotesInfo),
```

Comment:
Why not inlining this function?

Reply:
It's just a preference. I prefer to see the elements that are injected all at once and cut out the functions separately for easy maintenance. But I can inline this.

Reply:
No absolute truths, this is a matter of balance and personal preferences :slightly_smiling_face: 

IMHO, it's not necessary here and it create an indirection when reading/maintaining the code, I think I would prefer for you to inline if you don't mind (perhaps one day I'll change my mind on this :wink: )